keyboards/kprepublic/bm65hsrgb/rev2/rules.mk

```
@@ -0,0 +1,36 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+LTO_ENABLE = yes
+
+LAYOUTS = 65_ansi_blocker
+
+LTO_ENABLE = yes
+
+# RGB Matrix is required to support per-key LEDs connected to IS31FL3741.
+RGB_MATRIX_ENABLE = yes
```

Would probably be a good idea to add: 
```suggestion
RGB_MATRIX_ENABLE = yes
RGBLIGHT_SUPPORTED = no
```